### DISCUSSION TITLE
Brunch on Intel Core Ultra 7 thinkbook 14 2-in-1

### DISCUSSION THREAD

**Discussion by krishnarao-1** [sebanc/brunch#2070] [General]
Hi,
I have lenovo thinkbook with Intel Core 7 Ultra and I tried to install chrome os as image for dual boot first using the voxel image and later using a screebo image.
No luck and all I get is a black screen with no messages/nothing.
Which would be the best recovery for that CPU and any help would be very welcome.
Many thanks

**Comment by krishnarao-1** [comment]
I have tried volteer, rammus, rex, brya, nissa to no avail. Nothing works. Doesn't even boot to chrome os.

**Comment by Ken611116** [comment]
INTEL:
3th  (Ivy Bridge): "samus" , "reef"
4th  (Haswell): "rammus"
5th  (Broadwell): "rammus"
6th  (Skylake): "rammus" , "shyvana"
7th  (Kaby Lake): "shyvana" , "rammus"
8th  (Coffee Lake): shyvana" for Intel , "bobba" for Celeron.
9th  (Coffee Lake): "shyvana" for Intel , "bobba" for Celeron.
10th (Comet Lake):  "jinlon".
11th (Tiger Lake):  "voxel".
12th (Alder Lake):  "brya" , "brask"
12th (Alder Lake-N)  "craaskvin" , "craaskana" , "craasneto" ---->N100 N200 N305
xxth (Twin Lake)  "rynax" , "riven" , "rudriks" , "teliks" ----N150 , N250 N350 ....
13th (Raptor Lake): "brask".
14th (Raptor Lake): "brask".

Core Ultra 1 (Meteor Lake): "karis" , "screebo" ----100U
Core Ultra 2 (Lunar Lake): "karis" , "screebo" ----200V
Core Ultra 2 (Arrow Lake): "karis" , "screebo" ----200K , 200H , 200U

AMD
Ryzen: "gumboz".
